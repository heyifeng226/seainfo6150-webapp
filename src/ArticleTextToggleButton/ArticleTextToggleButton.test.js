import React from "react";
import ArticleTextToggleButton from "./ArticleTextToggleButton";


describe("ArticleTextToggleButton tests", () => {
    it("renders correctly", () => {
      const { container } = render(<ArticleTextToggleButton />);
      expect(container).toMatchSnapshot();
    });

    it("renders correctly when passed a buttonText prop", () => {
        const { container } = render(<ArticleTextToggleButton buttonText="showLess"/>);
        expect(container).toMatchSnapshot();
      });
  
  });