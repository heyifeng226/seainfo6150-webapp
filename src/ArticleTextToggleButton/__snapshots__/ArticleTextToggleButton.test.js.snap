// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleTextToggleButton tests renders correctly 1`] = `
<div>
  <button
    class="button"
  />
</div>
`;

exports[`ArticleTextToggleButton tests renders correctly when passed a buttonText prop 1`] = `
<div>
  <button
    class="button"
  >
    showLess
  </button>
</div>
`;
